addons/l10n_eu_service/tests/test_oss.py

```diff
@@ -41,6 +41,39 @@ def test_country_tag_from_belgium(self):
                 self.assertIn(expected_tag_id, oss_tag_id, f"{doc_type} tag from Belgian CoA not correctly linked")
 
 
+@tagged('post_install', 'post_install_l10n', '-at_install')
+class TestOSSSpain(AccountTestInvoicingCommon):
```

Comment:
Add a comment stating that this is for testing the case where the values in the mapping are tags
And another comment in `TestOSSBelgium` that it is the case for when the values are report lines
And another one for `TestOSSUSA` that it is for when OSS doesn't make sense